### PR TITLE
config: Revert "config: auto respond to lgtm (#334)"

### PR DESCRIPTION
**Pull request by tisonkun**
This reverts commit c2c11ec7a6915e6fb79647bb673bf3adaaf6841d.

Remove timely autorespond after july 1st and written before.

### CODE REVIEW

File: prow/config/plugins.yaml [hunk 1]
```diff
@@ -1110,47 +1040,6 @@ external_plugins:
...
-    - name: ti-community-label
-      events:
-        - issue_comment
-    - name: ti-community-label-blocker
-      events:
-        - pull_request
-    - name: ti-community-cherrypicker
-      events:
-        - pull_request
-        - issue_comment
```
Comment (Mini256) [inline prow/config/plugins.yaml:1148]: The revert operation contains parts that should not be deleted.

**Review by Mini256 — CHANGES_REQUESTED** [review]

File: prow/config/plugins.yaml (hunk 1 above)
Comment (tisonkun) [reply prow/config/plugins.yaml]: Thanks for your review! Yes I think it is a mistake when resolve conflict. Updated.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**